pkgs/development/tools/kustomize/default.nix

```diff
@@ -2,8 +2,8 @@
 
 buildGoModule rec {
   pname = "kustomize";
-  version = "3.1.0";
-  # rev is the 3.1.0 commit, mainly for kustomize version command output
+  version = "3.2.0";
+  # rev is the 3.2.0 commit, mainly for kustomize version command output
   rev = "95f3303493fdea243ae83b767978092396169baf";
```

Comment:
We need to change this commit hash too (otherwise the `kustomize version` output will be incoherent)